init_blocks needs to be 0 for kubernetes provider

When `init_blocks` is not set to 0 with kubernetes providers, funcx-endpoint raises the following exception:

```020-09-01 20:28:46.249 interchange:300 [INFO]  Scaling ...
2020-09-01 20:28:46.249 interchange:254 [ERROR]  Caught exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/funcx_endpoint/executors/high_throughput/interchange.py", line 252, in __init__
    self.load_config()
  File "/usr/local/lib/python3.6/site-packages/funcx_endpoint/executors/high_throughput/interchange.py", line 301, in load_config
    self.scale_out(self.config.provider.init_blocks)
  File "/usr/local/lib/python3.6/site-packages/funcx_endpoint/executors/high_throughput/interchange.py", line 802, in scale_out
    internal_block = self.config.provider.submit(launch_cmd, 1)
TypeError: submit() missing 1 required positional argument: 'task_type'
```

It should either raise a more informative exception to the user or ignore `init_blocks` for k8s.

--- a/funcx_endpoint/funcx_endpoint/providers/kubernetes/kube.py
+++ b/funcx_endpoint/funcx_endpoint/providers/kubernetes/kube.py
@@ -30,7 +30,7 @@ class KubernetesProvider(ExecutionProvider, RepresentationMixin):
     nodes_per_block : int
         Nodes to provision per block.
     init_blocks : int
-        Number of blocks to provision at the start of the run. Default is 1.
+        Number of blocks to provision at the start of the run. Default is 0.
     min_blocks : int
         Minimum number of blocks to maintain.
     max_blocks : int
@@ -77,7 +77,7 @@ class KubernetesProvider(ExecutionProvider, RepresentationMixin):
                  image: str,
                  namespace: str = 'default',
                  nodes_per_block: int = 1,
-                 init_blocks: int = 4,
+                 init_blocks: int = 0,
                  min_blocks: int = 0,
                  max_blocks: int = 10,
                  max_cpu: float = 2,

